Makefrag

```diff
@@ -58,12 +63,20 @@ mem: $(NAME).mem
 	@rm $*.binary.txt # Delete the temporary binary file
 
 # runs the program (generated .mem file) using fp-emu 
-run: $(NAME).mem
-	@$(EMU) +ispm=$^;
+run: $(NAME).mem $(CLIENT)
+ifeq ($(CLIENT_CMD),)
+	@$(EMU) +ispm=$^
+else
+	@$(CLIENT_CMD) &
+	@$(EMU) +ispm=$^ --client
+	@wait
+endif
```

Comment:
If the CLIENT variable is set, the emulator will be run with the `--client` argument. This intializes a server on the emulator which allows a client to connect. The client can send commands to set the pins on FlexPRET at specific clock cycles.